.readthedocs.yaml

```
@@ -1,22 +1,17 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
 version: 2
-
 build:
-  os: "ubuntu-22.04"
+  os: 'ubuntu-22.04'
   tools:
-    python: "3.10"
+    python: '3.11'
   jobs:
     post_checkout:
       - git fetch --unshallow
     pre_install:
       - git update-index --assume-unchanged docs/source/conf.py
-
+    post_install:
+      - 'curl -sSL https://raw.githubusercontent.com/pdm-project/pdm/main/install-pdm.py | python3 -'
+      - 'VIRTUAL_ENV=$(dirname $(dirname $(which python))) ~/.local/bin/pdm sync -dG docs'
```

```suggestion
      - '~/.local/bin/pdm use -f "$READTHEDOCS_VIRTUALENV_PATH"
      - '~/.local/bin/pdm sync -dG docs'
```
readthedocs has exposed the path so we don't need this hack.